ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/misc.py

```diff
@@ -8,6 +8,12 @@
 
 from ansible_collections.arista.avd.plugins.filter.convert_dicts import convert_dicts
 from ansible_collections.arista.avd.plugins.filter.natural_sort import natural_sort
+from ansible_collections.arista.avd.plugins.filter.range_expand import range_expand
+
+if TYPE_CHECKING:
+    from ansible_collections.arista.avd.plugins.plugin_utils.eos_designs_facts.eos_designs_facts import EosDesignsFacts
```

Comment:
```suggestion
if TYPE_CHECKING:
    from ansible_collections.arista.avd.plugins.plugin_utils.eos_designs_facts import EosDesignsFacts
```
There is already a TYPE_CHECKING block on line 19, you can move this there so it's cleaner :)

Reply:
fixed